aviary/mission/flight_phase_builder.py

```diff
@@ -376,89 +330,83 @@ def _extra_ode_init_kwargs(self):
+FlightPhaseBase._add_meta_data('fix_initial', val=True)
 
-EnergyPhase._add_meta_data('fix_duration', val=False)
+FlightPhaseBase._add_meta_data('fix_duration', val=False)
 
-EnergyPhase._add_meta_data('optimize_mach', val=False)
+FlightPhaseBase._add_meta_data('optimize_mach', val=False)
 
-EnergyPhase._add_meta_data('optimize_altitude', val=False)
+FlightPhaseBase._add_meta_data('optimize_altitude', val=False)
 
-EnergyPhase._add_meta_data('initial_bounds', val=(0., 100.), units='s')
+FlightPhaseBase._add_meta_data('initial_bounds', val=(0., 100.), units='s')
 
-EnergyPhase._add_meta_data('duration_bounds', val=(0., 100.), units='s')
+FlightPhaseBase._add_meta_data('duration_bounds', val=(0., 100.), units='s')
 
-EnergyPhase._add_meta_data(
+FlightPhaseBase._add_meta_data(
     'required_available_climb_rate', val=None, units='m/s',
     desc='minimum avaliable climb rate')
 
-EnergyPhase._add_meta_data(
+FlightPhaseBase._add_meta_data(
     'no_climb', val=False, desc='aircraft is not allowed to climb during phase')
 
-EnergyPhase._add_meta_data(
+FlightPhaseBase._add_meta_data(
     'no_descent', val=False, desc='aircraft is not allowed to descend during phase')
 
-EnergyPhase._add_meta_data('constrain_final', val=False)
-
-EnergyPhase._add_meta_data('input_initial', val=False)
-
-EnergyPhase._add_meta_data('initial_mach', val=None, units='unitless')
+FlightPhaseBase._add_meta_data('constrain_final', val=False)
 
-EnergyPhase._add_meta_data('final_mach', val=None, units='unitless')
+FlightPhaseBase._add_meta_data('input_initial', val=False)
 
-EnergyPhase._add_meta_data('initial_altitude', val=None, units='ft')
+FlightPhaseBase._add_meta_data('initial_mach', val=None, units='unitless')
 
-EnergyPhase._add_meta_data('final_altitude', val=None, units='ft')
+FlightPhaseBase._add_meta_data('final_mach', val=None, units='unitless')
 
-EnergyPhase._add_meta_data('throttle_enforcement', val=None)
+FlightPhaseBase._add_meta_data('initial_altitude', val=None, units='ft')
 
-EnergyPhase._add_meta_data('mach_bounds', val=(0., 2.), units='unitless')
+FlightPhaseBase._add_meta_data('final_altitude', val=None, units='ft')
 
-EnergyPhase._add_meta_data('altitude_bounds', val=(0., 60.e3), units='ft')
+FlightPhaseBase._add_meta_data('throttle_enforcement', val=None)
 
-EnergyPhase._add_meta_data('solve_for_distance', val=False)
+FlightPhaseBase._add_meta_data('mach_bounds', val=(0., 2.), units='unitless')
 
-EnergyPhase._add_meta_data('constraints', val={})
+FlightPhaseBase._add_meta_data('altitude_bounds', val=(0., 60.e3), units='ft')
 
-EnergyPhase._add_initial_guess_meta_data(
-    InitialGuessIntegrationVariable(),
-    desc='initial guess for initial time and duration specified as a tuple')
+FlightPhaseBase._add_meta_data('solve_for_distance', val=False)
 
-EnergyPhase._add_initial_guess_meta_data(
-    InitialGuessState('distance'),
-    desc='initial guess for horizontal distance traveled')
+FlightPhaseBase._add_meta_data('constraints', val={})
 
```

Comment:
This diff is kind of confusing, but did we lose a couple of initial guesses?  Initial time+duration and distance.